misk-slack/api/misk-slack.api

```diff
@@ -1,27 +1,27 @@
-public final class misk/slack/RealSlackClient : misk/slack/SlackClient {
```

Comment:
This denotes a breaking change (looks like from the package name changing), how many existing usages are there? What is the migration plan?

Alternatively, I'd be in favour of changing the package/dir back to slack. The API suffix doesn't add much.

Reply:
I'm going to change this back to slack. The `slack-api` refers to the new web API being added, but the new and old APIs seemed to collide.